### PR TITLE
Added stats view

### PR DESCRIPTION
**Pull request by mpodgorny**
Also, fool proofing `update_stats` call in game.py - this should no longer throw exception in current game handling.

### CODE REVIEW

File: backend/src/routes/stats.py [hunk 1]
```diff
@@ -31,12 +31,12 @@
 
 def get_current_top():
     players = UserStats.objects.order_by("-games_won").limit(5)
-    top5_dict = {}
-    for i, player in enumerate(players):
+    top5_dict = []
+    for player in enumerate(players):
```
Comment (pawelrubin) [inline backend/src/routes/stats.py]: Unnecessary `enumerate`.

File: backend/src/routes/game.py [hunk 2]
```diff
@@ -53,7 +53,8 @@
             await check_timers(websocket, username, game)
             await game.push({"code": "players", "players": game.get_players_hp()})
         except (WebSocketDisconnect, ConnectionClosedError):
-            update_stats(game.players_data[username], username, False)
+            if username in game.players_data:
```
Comment (pawelrubin) [inline backend/src/routes/game.py]: It hides a serious bug in our code, imo you should leave a comment on that issue.

By the way, in Python we should use EAFP (easier to ask for forgiveness than permission) coding style. 
This `if` is an anti-pattern. You can read more [here](https://docs.quantifiedcode.com/python-anti-patterns/readability/asking_for_permssion_instead_of_forgiveness_when_working_with_files.html)

So instead you should use `try/except KeyError`

File: frontend/src/components/stats/Stats.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    const statistics = async () => {
+      try {
+        fetch(`/api/stats`)
+          .then(res => res.json())
+          .then(res => updateStats(res));
+      } catch (e) {
+        console.log("Stats error");
+      }
+    };
+    statistics();
```
Comment (pawelrubin) [inline frontend/src/components/stats/Stats.js]: If you are using this function in one place only, why declare it?

File: frontend/src/components/stats/Stats.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+import GoBackButton from "../GoBackButton/GoBackButton";
+import LoggedContext from "../../contexts/LoggedContext";
+
+const EmptyStats = { top5: [], player_stats: {} };
+const roman = ["I", "II", "III", "IV", "V"];
+
+const Stats = () => {
+  const history = useHistory();
+  const isLogged = useContext(LoggedContext);
+  const [stats, updateStats] = useState(EmptyStats);
```
Comment (pawelrubin) [inline frontend/src/components/stats/Stats.js]: When using `useState()`, you should follow the naming convention:
```JavaScript
const [foo, setFoo] = useState();
```

File: backend/src/routes/stats.py [hunk 5]
```diff
@@ -31,12 +31,12 @@
...
-    top5_dict = {}
-    for i, player in enumerate(players):
+    top5_dict = []
+    for player in players:
         player_json = {}
         player_json["username"] = player.username
         player_json["games_won"] = player.games_won
-        top5_dict[i] = player_json
+        top5_dict.append(player_json)
     return top5_dict
```
Comment (pawelrubin) [inline backend/src/routes/stats.py]: Try using list comprehension.

**Review by SzymonGlab — APPROVED** [review]

**Review by karolszymonczyk — APPROVED** [review]

**Pull request merged**